cmd/juju/model/exportbundle.go

```diff
@@ -0,0 +1,117 @@
+// Copyright 2018 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+package model
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/juju/cmd"
+	"github.com/juju/errors"
+	"github.com/juju/gnuflag"
+
+	"github.com/juju/juju/api/bundle"
+	"github.com/juju/juju/cmd/modelcmd"
+	"github.com/juju/juju/cmd/output"
+)
+
+// NewExportBundleCommand returns a fully constructed export bundle command.
+func NewExportBundleCommand() cmd.Command {
+	return modelcmd.Wrap(&exportBundleCommand{})
+}
+
+type exportBundleCommand struct {
+	modelcmd.ModelCommandBase
+	out cmd.Output
+	api ExportBundleModelAPI
```

Comment:
Please have it as func to access api instead of the api directly, something like
```
newAPIFunc func() (ExportBundleAPI, error)
```
There are commands that have similar accessors.

Reply:
done.